go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/cert-manager/csi-lib
 
-go 1.17
+go 1.18
 
 require (
-	github.com/cert-manager/cert-manager v1.8.0
-	github.com/container-storage-interface/spec v1.4.0
-	github.com/go-logr/logr v1.2.0
-	github.com/kubernetes-csi/csi-lib-utils v0.9.1
+	github.com/cert-manager/cert-manager v1.8.1
+	github.com/container-storage-interface/spec v1.6.0
+	github.com/go-logr/logr v1.2.3
+	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/net v0.0.0-20220107192237-5cfca573fb4d
-	google.golang.org/grpc v1.43.0
+	golang.org/x/net v0.0.0-20220615171555-694bf12d69de
+	google.golang.org/grpc v1.47.0
 	k8s.io/apimachinery v0.23.4
```

Comment:
Didn't upgrade `k8s.io` vendor as it would have conflicts with `github.com/cert-manager/cert-manager` on `gnostic` dependency.